website/snippets/core-versions-table.md

```diff
@@ -3,7 +3,7 @@
 |                      dbt Core                                 | Initial release |      Support level and end date      |
 |:-------------------------------------------------------------:|:---------------:|:-------------------------------------:|
 | [**v1.8**](/docs/dbt-versions/core-upgrade/upgrading-to-v1.8) | May 9 2024    | <b>Active Support &mdash; May 8, 2025</b>                    |
-| [**v1.7**](/docs/dbt-versions/core-upgrade/upgrading-to-v1.7) | Nov 2, 2023   | <div align="left">**dbt Core and dbt Cloud Developer & Team customers:** Critical Support until Nov 1, 2024 <br /> **dbt Cloud Enterprise customers:** Critical Support until further notice <sup>1</sup></div> | 
+| [**v1.7**](/docs/dbt-versions/core-upgrade/upgrading-to-v1.7) | Nov 2, 2023   | <div align="left">**dbt Core and dbt Cloud Developer & Team customers:** End of Life <br /> **dbt Cloud Enterprise customers:** Critical Support until further notice <sup>1</sup></div> | 
```

Comment:
@jtcohen6 Should we update [the note](https://github.com/dbt-labs/docs.getdbt.com/pull/6400/files#diff-9de7657ba12df571df5a84e8022f4c95e94a2fea26b458aa3412627ccdfc6006R18) after the table?

Maybe to something like:
> "Versionless" is now required for the Developer and Teams plans on dbt Cloud. Accounts on older dbt versions will be migrated to "Versionless."

Reply:
Added here: https://github.com/dbt-labs/docs.getdbt.com/pull/6400/commits/1d0530b6870b728aa4e87313c629d818d108cb6d